app/api/app_last_active.ts

```diff
@@ -1,18 +1,28 @@
 import { EnvironmentName, getEnvironment } from '../environment'
 import { StorageAPI } from './storage'
 
-const KEY = 'APP_LAST_ACTIVE.timestamp'
+const KEYS = {
+  force: 'APP_LAST_ACTIVE.force',
+  timestamp: 'APP_LAST_ACTIVE.timestamp'
+}
 
 async function set (): Promise<void> {
-  return await StorageAPI.setItem(KEY, `${Date.now()}`)
+  return await StorageAPI.setItem(KEYS.timestamp, `${Date.now()}`)
+}
+
+async function removeForceAuth (): Promise<void> {
+  return await StorageAPI.removeItem(KEYS.force)
 }
 
 async function forceRequireReauthenticate (): Promise<void> {
-  return await StorageAPI.setItem(KEY, `${Date.now() - getTimeoutPeriod()}`)
+  return await StorageAPI.setItem(KEYS.force, 'TRUE')
```

Comment:
Need a higher priority persistent flag. The timestamp tends to get overwrite as the component lifecycle is not very controllable. (The focus out callback reset my "force" timestamp when I programmatically exit app)